.github/workflows/push-validation.yml

```
@@ -12,7 +12,7 @@ jobs:
     - name: Set up JDK 1.8
       uses: actions/setup-java@v1
       with:
-        java-version: 1.8
+        java-version: 1.11
```

According to https://github.com/actions/setup-java I would think this should rather be just "11":

```suggestion
        java-version: 11
```